packages/contracts-bedrock/deploy-config/mainnet.json

```
@@ -12,25 +13,24 @@
   "p2pSequencerAddress": "0x15d34AAf54267DB7D7c367839AAf71A00a2C6A65",
   "batchInboxAddress": "0xff00000000000000000000000000000000000010",
   "batchSenderAddress": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
-  "l2OutputOracleSubmissionInterval": 20,
-  "l2OutputOracleStartingTimestamp": 1679069195,
-  "l2OutputOracleStartingBlockNumber": 79149704,
+  "l2OutputOracleSubmissionInterval": 1800,
+  "l2OutputOracleStartingTimestamp": 0,
+  "l2OutputOracleStartingBlockNumber": 0,
   "l2OutputOracleProposer": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "l2OutputOracleChallenger": "0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC",
-  "finalizationPeriodSeconds": 2,
-  "proxyAdminOwner": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
-  "baseFeeVaultRecipient": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
-  "l1FeeVaultRecipient": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
-  "sequencerFeeVaultRecipient": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
+  "l2OutputOracleChallenger": "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266",
+  "finalizationPeriodSeconds": 3600,
+  "baseFeeVaultRecipient": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+  "l1FeeVaultRecipient": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
+  "sequencerFeeVaultRecipient": "0x70997970C51812dc3A010C7d01b50e0d17dc79C8",
   "governanceTokenName": "Optimism",
   "governanceTokenSymbol": "OP",
-  "governanceTokenOwner": "0x90F79bf6EB2c4f870365E785982E1f101E93b906",
-  "l2GenesisBlockGasLimit": "0x1c9c380",
+  "governanceTokenOwner": "0x5C4e7Ba1E219E47948e6e3F55019A647bA501005",
+  "l2GenesisBlockGasLimit": "0x017D7840",
```

Possible footgun if the deployment doesn't like leading zeros in the hex like RPC endpoints. The old value has an uneven number of nibbles: https://github.com/ethereum-optimism/optimism/blob/e43cd979b63c9709d7c33b4fb5f5b2d0c6e7b462/packages/contracts-bedrock/deploy-config/mainnet.ts#L40

```suggestion
  "l2GenesisBlockGasLimit": "0x17D7840",
```